### PR TITLE
Update if 0 3 5

### PR DESCRIPTION
**Pull request by alexgraham-da**
Update to `daml_dit_if-0.3.5`

### CODE REVIEW

File: dabl-meta.yaml [hunk 1]
```diff
@@ -4,13 +4,14 @@
 catalog:
     name: dabl-integration-symphony
     group_id: com.digitalasset
-    version: 0.2.0
+    version: 0.2.1
     short_description: Symphony
-    description: Symphony Send and Receive
+    description: Send and Receive messages and form elements through the Symphony messaging app
```
Comment (mschaef-da) [inline dabl-meta.yaml:9]: Does this fit on screen?

File: requirements.txt [hunk 2]
```diff
@@ -1,9 +1,4 @@
-./lib/PyYAML-5.3.1.tar.gz
 ./lib/python-json-logger-0.1.11.tar.gz
 ./lib/yattag-1.12.2.tar.gz
-dazl==7.0.2
-yarl==1.5.1
-multidict==4.7.6
-daml_dit_api==0.3.0
-daml_dit_if==0.3.0
+daml_dit_if==0.3.5
```
Comment (mschaef-da) [inline requirements.txt:3]: This all makes me unreasonably happy

**Review by mschaef-da — APPROVED** [review]
LGTM, provided the text will fit on the screen.

File: dabl-meta.yaml (hunk 1 above)
Comment (mschaef-da) [reply dabl-meta.yaml:9]: Thinking out loud, it would be nice if we could include links in these descriptions...

Comment (alexgraham-da) [reply dabl-meta.yaml:9]: Yeah, I was wanting to be able to put in a link to Symphony's website a while ago when writing that blurb.

Here's the description in context (after changing "through" to "via"):
Screen Shot 2021-02-04 at 10 24 06 AM

A bit longer than the others, but I think having "Symphony messaging app" is useful for those who might not know what Symphony is, vs. Slack which is ubiquitous.

Let me know what you think.

**Pull request merged**